### DISCUSSION TITLE
Rendering HTML on Table unintendedly hides some of the column's data

### DISCUSSION THREAD

**Discussion by andro42** [apache/superset#27444] [Ideas]
## Description

I have a problem with column being rendered as html in Table visualisation. This is not wanted. I want to display text from database, without being rendered as html. In my case, there is text  in my databases table "...< [some.email@domain.com] >..." in my column, which then disappears in table chart. How can I disable htlm rendering (for this column, or for this chart, or for whole application)? Using docker and superset version "x-superset-image: &superset-image apache/superset:${TAG:-latest-dev}" at the time of this posting, which shows "About Version: 0.0.0-dev"

## Design input
[`design:suggest`]: Maybe add an option not to render HTML in table (column). Redash has this option.

**Comment by carloscamposcoelho** [comment]
Same issue here, on chart table visualization it is hiding the text between <...>. On the dataset and query editor, it is ok.

**Comment by rusackas** [comment]
Moving this to a Discussion since it's not a bug, but note that a similar feature (for SQL Lab) is being discussed on the dev@ mailing list presently: https://lists.apache.org/thread/crykcog9p37bx9psypk33ykkbmyvwwvs
